operator/v1/tests/consoles.operator.openshift.io/AAA_ungated.yaml

```diff
@@ -296,3 +296,138 @@ tests:
               visibility:
                 state: Enables
       expectedError: "spec.customization.capabilities[0].visibility.state: Unsupported value: \"Enables\": supported values: \"Enabled\", \"Disabled\""
+    - name: Should allow creating custom logos for Dark and Light themes
+      initial: |
+        apiVersion: operator.openshift.io/v1
+        kind: Console
+        spec:
+          customization:
+            customLogos:
+              - type: Masthead
+                themes:
+                  - type: Dark
```

Comment:
Are themes typed, or named? Should this be `-name: Dark`?

Reply:
typed

Reply:
I think this conversation moved towards the term `mode` in the slack thread